### PR TITLE
Fix vampire revive

### PR DESCRIPTION
**Pull request by ghost**
:cl: steamp0rt
fix: Vampire revive should be able to work again
tweak: Vampire revive now regenerates limbs and organs.
/:cl:

### CODE REVIEW

File: hippiestation/code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -327,18 +327,18 @@
...
 	user.visible_message("<span class='warning'>[user] reanimates from death!</span>", "<span class='notice'>We get back up.</span>")
-	user.fully_heal(TRUE)
-
-
-
+	var/list/missing = user.get_missing_limbs()
+	if(missing.len)
+		playsound(user, 'sound/magic/demon_consume.ogg', 50, 1)
+		user.visible_message("<span class='warning'>Shadowy matter takes the place of [user]'s missing limbs as they reform!</span>")
+		user.regenerate_limbs(0, list(BODY_ZONE_HEAD))
```
Comment (Carbonhell) [inline hippiestation/code/game/gamemodes/vampire/vampire_powers.dm:339]: mfw vampires can revive but lings can not

Comment (ghost) [reply hippiestation/code/game/gamemodes/vampire/vampire_powers.dm:339]: vampires are unholy monsters

lings are just space assholes

Comment (Manly-Dwarf) [reply hippiestation/code/game/gamemodes/vampire/vampire_powers.dm:339]: pain

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
Did you test this?

**Comment by Eagle-Eyess** [comment]
+1

**Comment by ghost** [comment]
i just copypasted from ling revive lol

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**